$NetBSD: patch-ag,v 1.1 2009/08/26 21:44:43 he Exp $

Doing a static initialization causes this to be a local variable
in this object file.  This does not override the library version
which is stored in a variable in the math library(!)

Instead, we need to initialize _LIB_VERSION in code, ref.
patch-af.

--- config/gen/platform/netbsd/math.c.orig	2009-07-08 02:19:09.000000000 +0200
+++ config/gen/platform/netbsd/math.c
@@ -22,14 +22,8 @@ math stuff
 
 */
 
-/*
- * force atan2() to use IEEE behavior
- */
-
 #include <math.h>
 
-_LIB_VERSION_TYPE _LIB_VERSION = _IEEE_;
-
 #if DOUBLE_SIZE == 2 * INT_SIZE
 /*
 
